net/acme-common/files/acme.init

```diff
@@ -163,11 +168,15 @@ service_triggers() {
 		/etc/init.d/acme renew
 }
 
-renew() {
+load_and_run() {
 	trap cleanup EXIT
 
 	config_load acme
 	config_foreach load_globals acme
 
 	config_foreach get_cert cert
 }
+
+renew() {
+	rc_procd load_and_run
```

Comment:
AFAICT, this causes `rc_procd` to create the service with an "add" method. Not sure exactly how procd service tracking works, but what happens if/when `renew()` is called multiple times?